Lib/defcon/objects/font.py

```diff
@@ -618,6 +618,8 @@ def updateGlyphOrder(self, addedGlyph=None, removedGlyph=None):
         order = self.glyphOrder
         index = None
         if removedGlyph is not None:
+            if removedGlyph == addedGlyph:
+                return
```

Comment:
hm. what if removedGlyph == addedGlyph, and they are not in the glyphOrder? Then you exit early without appending the addedGlyph...